index.html

```diff
@@ -78,6 +78,17 @@ <h2>Add New Recipe</h2>
 
     <div id="recipe-container"></div>
 
+    <div id="image-container"></div>
+
+    <div id="recipe-container"></div>
+
+    <input type="text" id="searchInput" placeholder="Search by title" />
```

Comment:
It would be better to have the search input and button in a `form` element and use the [`submit` event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) so the user could also press "enter" to search for a recipe.